### PR TITLE
create script.js file

### PR DESCRIPTION
**Pull request by neoThobela**
<p>this is my js file for my chatbot</p>

### CODE REVIEW

**Comment by kgaugelomokgalaka** [comment]
the code is good , easy to be read by the other person
overall its a good code

**Comment by ManeoMalatsi** [comment]
I like your colours and chatbot, please show how to insert the icon

**Comment by NthabisengLekolwana** [comment]
i love it, its friendly.

**Review by NthabisengLekolwana — CHANGES_REQUESTED** [review]
You did not update your read me file after adding the chatbot to your website

File: script.js [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+    "education" : ["I have a BSC in computer science and mathematical sciences, i majored in Computer science and statistics"],
+    "experience" : [" I have experience as a Technical Assistant, supporting practical lessons and enforcing lab rules. As a Lab Assistant, I contributed to software development, collaborating with senior engineers and engaging in continuous learning. Most recently, as an Associate Software Engineer, I educated clients on financial products and delivered exceptional service, currently trainig as a Cobol developer at capaciti"],
+    "default": [
+        "I'm not sure I understand that.",
+        "Could you please rephrase?",
+        "Hmm, I don't quite understand.",
+        "please contact Neo for more information at Tell: 083 281 6746 or email me at neo.thobela@capaciti.org.za"
+    ]
+    
+};
```
Comment (XolaniMba) [inline script.js:56]: @neoThobela  The chatbot responses are really good!

**Review by XolaniMba — APPROVED** [review]

**Review by kgaugelomokgalaka — COMMENTED** [review]
your code look good 
please just reduce the amount of comments in your css file

**Review by ManeoMalatsi — APPROVED** [review]

**Comment by neoThobela** [comment]
@NthabisengLekolwana  thank you ive updated it.

**Review by NthabisengLekolwana — APPROVED** [review]

**Comment by neoThobela** [comment]
> your code look good please just reduce the amount of comments in your css file

@kgaugelomokgalaka done thank you

**Pull request merged**